server/app.js

```diff
@@ -15,6 +15,77 @@ app.engine(
   })
 );
 
+let pg = require("pg");
+
+var knex = require("knex")({
+  client: "pg",
+  connection: {
+    host: "127.0.0.1",
+    user: "cyf",
+    password: "password",
+    database: "orula",
+    port: 5432
+  }
+});
+
+// get '/classes'
+app.get("/classes", (req, res) => {
+  const sqlStatement = "select * from classes";
+  knex.raw(sqlStatement).then(data => {
+    res.json(data);
+  });
+});
+const callBack = (error, classes) => {
+  if (error) {
+    console.error(error);
+    res.sendStatus(500);
+  } else {
+    res.json(classes);
+  }
+};
+// get '/classes/class_id/:id'
+app.get("/classes/class_id/:id", (req, res) => {
+  const sqlStatement = `select * from classes where class_id =${req.params.id}`;
+  knex.raw(sqlStatement).then(data => {
+    res.json(data);
+  });
+});
+// post '/classes/'
+app.post("/classes", function(req, res) {
+  const body = req.body;
+  const sqlStatement = `INSERT INTO classes
+  (class_id,name, created_at, updated_at) 
+  VALUES ("${body.class_id}", "${body.name}", "${body.created_at}", "${
+    body.updated_at
+  }");`;
+  knex.raw(sqlStatement).then(data => {
+    //console.log(req.body)
+    res.send("successfully classes added");
+  });
+});
+// put '/classes/class_id/:id'
+app.put("/classes/class_id/:id", function(req, res) {
```

Comment:
you don't need class_id in the route. For [REST](https://searchmicroservices.techtarget.com/definition/RESTful-API), you just call it `/classes/:id` - no need for the word "class_id" in all these routes.